### PR TITLE
Core stripe integrations

### PR DESCRIPTION
**Pull request by tmclean15**
Added payment route handlers, Fund and PaymentMethod db models, added columns and relationships to User and Project model, added payment processing functions.

### CODE REVIEW

File: client/src/pages/Payment.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { Redirect } from "react-router-dom";
+
+const stripePromise = loadStripe(process.env.REACT_APP_STRIPE_PUBLISHABLE_KEY);
+
+function Payment(props) {
+  const handlePayment = async (event) => {
+    const sessionId = await axios
+      .post("/api/v1/payment/session", {
+        success_url:
+          "http://localhost:3000/payment?session_id={CHECKOUT_SESSION_ID}&redirect=/project",
```
Comment (bonnieli) [inline client/src/pages/Payment.js]: i'm guessing the full URL is needed for success and cancel URL, although it'll be best to store the root domain `http://localhost:3000` (for development) in an env var

File: client/src/pages/Payment.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    const stripe = await stripePromise;
+
+    const { error } = await stripe.redirectToCheckout({ sessionId });
+
+    if (error) console.log(error);
+  };
+
+  useEffect(() => {
+    const sessionId = query.parse(props.location.search).session_id;
```
Comment (bonnieli) [inline client/src/pages/Payment.js]:
```suggestion
    const {session_id: sessionId, redirect: redirectPath} = query.parse(props.location.search);
```

File: server/api/ping_handler.py [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
-@ping_handler.route('/api/v1/ping', methods=['POST'])
+@ping_handler.route('/api/v1/ping', methods=['GET'])
 def ping():
-    if request.method == 'POST':
-        body = json.loads(request.get_data())
-        if body['teamName'] in Config.TEAM_NAME.split(', '):
-            return jsonify({'response': "{} is now part of the team".format(body['teamName'])}), 200
-        else:
-            return jsonify({'response': "{} is not part of the team, change your .env".format(body['teamName'])}), 400
+    funds = payout_project(3)
```
Comment (bonnieli) [inline server/api/ping_handler.py:10]: is this just hardcoded demo project_id `3` for now?

Comment (bonnieli) [reply server/api/ping_handler.py:10]: oh wait yeah this is under /ping route nvm

**Review by bonnieli — COMMENTED** [review]
nice!

File: client/src/pages/Payment.js (hunk 1 above)
Comment (tmclean15) [reply client/src/pages/Payment.js]: Agreed, I will make that change.

File: server/api/ping_handler.py (hunk 3 above)
Comment (tmclean15) [reply server/api/ping_handler.py:10]: Yes, sorry I should have made this more clear. I was using the ping route to test outputs

File: client/src/pages/Payment.js (hunk 1 above)
Comment (tmclean15) [reply client/src/pages/Payment.js]: I think `window.location.origin` should do it

File: client/src/App.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ import Signup from "./pages/Signup";
 import Login from "./pages/Login";
 import Project from "./pages/Project";
 import CreateProject from "./pages/CreateProject";
+import Payment from "./pages/Payment";
 import Upload from "./pages/Upload";
```
Comment (MalcolmAG) [inline client/src/App.js]:
```suggestion
```
This page is deleted

File: client/src/App.js [hunk 5]
```diff
@@ -26,6 +27,7 @@ function App() {
           <Route path="/login" component={Login} />
           <Route path="/upload" component={Upload} />
```
Comment (MalcolmAG) [inline client/src/App.js]:
```suggestion
```
Same with this one

**Review by MalcolmAG — CHANGES_REQUESTED** [review]
The /payment route works on my machine no problems. Just request small changes

**Review by MalcolmAG — APPROVED** [review]
Looks good

**Pull request merged**